README.md

```diff
@@ -159,6 +159,7 @@ To enable debug info, run your program with environment flags
 - **deleteTicket(id, callback)** - Delete a ticket by its id
 - **restoreTicket(id, callback)** - Restore a ticket by its id
 - **listAllTickets(filter, callback)** - List All Tickets, check list of [filters](http://developer.freshdesk.com/api/#list_all_tickets)
+- **filterTickets(filter, callback)** - Filter tickets, based on ticket [fields](https://developer.freshdesk.com/api/#filter_tickets)
```

Comment:
And the name `filter` is not appropriate for this parameter. You should use `query`, because in other methods `filter` is an object, and here `query` - if a formatted string.